themes/navy/layout/index.swig

```diff
@@ -57,3 +57,23 @@
     </div>
   </div>
 </div>
+
+<div id="plugin-wrap">
+  <div class="wrapper">
+    <div class="inner plugin-wrapper">
+      <div class="plugin-thumbnail">
+        <div class="plugin-thumbnail-container">
+          <img src="/img/carbon.svg">
```

Comment:
any idea on the screenshot? or we just embed a zoomed-out screenshot of https://hexo.io/plugins/ ?

Reply:
How about free-to-use illustrations or equivalent?

e.g) followings

* https://isometric.online/ ([MIT](https://isometric.online/license/))
* https://undraw.co/ ([free-to-use](https://undraw.co/license))